code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -550,6 +550,13 @@
 	required_reagents = list("mindbreaker" = 1, "acetone" = 1, "inaprovaline" = 1)
 	result_amount = 3
 
+/datum/chemical_reaction/koispasteclean
+	name = "Filtered K'ois"
+	id = "koispasteclean"
+	result = "koispasteclean"
+	required_reagents = list("koispaste" = 1, "water" = 1, "ammonia" = 1)
+	result_amount = 3

```

Comment:
so the water and ammonia become part of the k'ois?

Reply:
yes, it would be like mixing a new drink

Reply:
That doesn't make sense if they're supposed to be 'cleaning' it.